test/runtests.jl

```diff
@@ -101,6 +101,66 @@ else  # run HiGHS tests
         @test r["ElectricStorage"]["size_kwh"] ≈ 83.3 atol=0.1
     end
 
+    @testset "Solar and ElectricStorage w/ zero ElectricStorage cost constants" begin
```

Comment:
@toddleif  would it be worth including a test in which the cost constant is provided, but the BESS max_kw is 0, and checking to see if the cost constant gets added when BESS size is 0?

Reply:
That's a great idea, thank you. I added a test for that scenario.